### PR TITLE
Only upload the service on one builder.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -26,6 +26,9 @@ jobs:
         include:
           - os: ubuntu-latest
             run_supabase_tests: true
+          - os: ubuntu-latest
+            run_supabase_tests: false
```
Comment (kasperl) [inline .github/workflows/ci.yml:30]: Why is this necessary?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -26,6 +26,9 @@ jobs:
...
+            upload-service: true
```
Comment (kasperl) [inline .github/workflows/ci.yml:31]: Remind me (again): Why is this not set to false at the top-level matrix like `run_supabase_tests`?

**Review by kasperl — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (floitsch) [reply .github/workflows/ci.yml:30]: We are choosing to set the flag for the existing combination ubuntu-latest/false.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (floitsch) [reply .github/workflows/ci.yml:31]: Because we would then add a new combination and have two 'ubuntu-latest/false'. One with upload-service set, and one without.
